src/Libraries/Microsoft.Extensions.Http.Resilience/Hedging/HedgingEndpointOptions.cs

```diff
@@ -20,33 +21,40 @@ public class HedgingEndpointOptions
     /// Gets or sets the bulkhead options for the endpoint.
     /// </summary>
     /// <remarks>
-    /// By default it is initialized with a unique instance of <see cref="HttpBulkheadPolicyOptions"/> using default properties values.
+    /// By default it is initialized with a unique instance of <see cref="HttpRateLimiterStrategyOptions"/> using default properties values.
     /// </remarks>
     [Required]
     [ValidateObjectMembers]
-    public HttpBulkheadPolicyOptions BulkheadOptions { get; set; } = new();
+    public HttpRateLimiterStrategyOptions RateLimiterOptions { get; set; } = new HttpRateLimiterStrategyOptions
+    {
+        StrategyName = StandardHedgingStrategyNames.RateLimiter
+    };
 
     /// <summary>
     /// Gets or sets the circuit breaker options for the endpoint.
     /// </summary>
     /// <remarks>
-    /// By default it is initialized with a unique instance of <see cref="HttpCircuitBreakerPolicyOptions"/> using default properties values.
+    /// By default it is initialized with a unique instance of <see cref="HttpCircuitBreakerStrategyOptions"/> using default properties values.
     /// </remarks>
     [Required]
     [ValidateObjectMembers]
-    public HttpCircuitBreakerPolicyOptions CircuitBreakerOptions { get; set; } = new();
+    public HttpCircuitBreakerStrategyOptions CircuitBreakerOptions { get; set; } = new HttpCircuitBreakerStrategyOptions
+    {
+        StrategyName = StandardHedgingStrategyNames.CircuitBreaker
+    };
 
     /// <summary>
-    /// Gets or sets the options for the timeout policy applied per each request attempt.
+    /// Gets or sets the options for the timeout resilience strategy applied per each request attempt.
     /// </summary>
     /// <remarks>
-    /// By default it is initialized with a unique instance of <see cref="HttpTimeoutPolicyOptions"/>
-    /// using a custom <see cref="TimeoutPolicyOptions.TimeoutInterval"/> of 10 seconds.
+    /// By default it is initialized with a unique instance of <see cref="HttpTimeoutStrategyOptions"/>
+    /// using a custom <see cref="TimeoutStrategyOptions.Timeout"/> of 10 seconds.
     /// </remarks>
     [Required]
     [ValidateObjectMembers]
-    public HttpTimeoutPolicyOptions TimeoutOptions { get; set; } = new()
+    public HttpTimeoutStrategyOptions TimeoutOptions { get; set; } = new()
     {
-        TimeoutInterval = _timeoutInterval,
+        Timeout = _timeoutInterval,
```

Comment:
```suggestion
        Timeout = TimeSpan.FromSeconds(10),
```

which means you don't need _timeoutInterval field, saving 8 bytes :-)